docs/development/api_updates.md

```diff
@@ -10,18 +10,14 @@ that directory directly (the unversioned API) and in the versioned subdirectorie
 
 ## Updating the generated API code
 
-You will need a few tools from kubernetes (these will likely be moved to a shared repo soon):
+To generate the API code simply run
 
 ```
-go get k8s.io/kubernetes
-cd ${GOPATH}/src/k8s.io/kubernetes
-git checkout master
-git pull
+make apimachinery
```

Comment:
As the documentation I deleted said apimachinery was to be extracted to a separate repository [which has since been done](https://github.com/kubernetes/apimachinery/) and now you no longer have to make kubernetes to generate the code.

Reply:
You are absolutely correct, don't know how I missed this.